randovania/game_connection/connector/prime1_remote_connector.py

```diff
@@ -112,6 +141,16 @@ async def _patches_for_pickup(self, provider_name: str, pickup: PickupEntry, inv
             if delta == 0:
                 continue
 
+            if item.extra.get("unk2_bitmask_value"):
+                patches.append(
+                    all_prime_dol_patches.adjust_item_amount_and_capacity_patch(
+                        self.version.powerup_functions,
+                        self.version.game,
+                        self.game.resource_database.get_item("Unknown2").extra["item_id"],
+                        (item.extra["item_id"] - 1000),
+                    )
```

Comment:
IIRC ulti mentioned that instead of writing to unknown2 directly, we should just write to the normal item IDs, so lets try that.
(also the delta being the item id  1000 is a bit weird here shouldnt it be the bitmask value?)

Reply:
technically unk2_bitmask is only used for checking if you have the pickup as you cannot use haspickup call dynamically from the connector

Reply:
also it becomes :
```py
                    all_prime_dol_patches.adjust_item_amount_and_capacity_patch(
                        self.version.powerup_functions,
                        self.version.game,
                        item.extra["item_id"],
                        delta,
                    )```